ChangeLog

```diff
@@ -1,6 +1,39 @@
 ??/??/???? - version 1.7.1
 	* Added companion matrix of a polynomial `companionmx p` and the
 	  theorems: companionmxK, map_mx_companion and companion_map_poly
+	* Extended generic theory about homomorphisms and monomophismes.
```

Comment:
I think that this line might be misleading (see below). It is not clear what a "homomorphism" is. And the `mono` in the {mono...} notation does not stand for monomorphism but for monotonic. If I am correct, a monomorphism is usually something else, e.g. an injective homomorphism. Plus in English there is no final 'e'.

Reply:
Sure, but I still think that the wording you added to the ChangeLog is misleading. It is not generic: it is about homo/mono morphism of *ordered* structures. See e.g. [homomorphism](https://en.wikipedia.org/wiki/Homomorphism). E.g. there is no equational theory preserved here. Adding that this is about order would address my comment. 

I have no complain about the name of the constants, and the comments in ssrfun are clear that they are about stability/morphism/homomorphism *for unary and binary relations*.

Reply:
> it is about homo/mono morphism of ordered structures

oh, right!